### PR TITLE
Feat: add src_account_type and dst_account_type options to mirror difference account type.

### PR DESCRIPTION
**Pull request by snowyu**
* feat: add src_account_type and dst_account_type options to mirror difference account type
* Fix: Add a 2 second delay after the repo is created to prevent errors in reporting that the repo does not exist

### CODE REVIEW

**Comment by Yikun** [comment]
Wow，感谢贡献，这个确实是一个合理的诉求，用于组织和用户的同步。

我这周内看下。有验证过没？比如gitee组织到github用户，github用户到gitee组织。

**Comment by snowyu** [comment]
confimed on github user to github org.

File: README_en.md [hunk 1]
```diff
@@ -34,7 +36,9 @@ More than [100+](https://github.com/search?p=2&q=hub-mirror-action+%22account_ty
 - `dst_token` the API token to create non-existent repo, You can get Github token in [here](https://github.com/settings/tokens), and the Gitee in [here](https://gitee.com/profile/personal_access_tokens).
 
 #### Optional
-- `account_type` (optional) default is `user`, the account type of src and dst account, can be set to `org` or `user`，only support mirror between same account type (that is "org to org" or "user to user").
+- `account_type` (optional) default is `user`, the account type of src and dst account, can be set to `org` or `user`，only support mirror between same account type (that is "org to org" or "user to user"). if u wanna mirror difference account type, use the `src_account_type` and `dst_account_type` please.
+- `src_account_type` (optional) default is `user`, the account type of src account, can be set to `org` or `user`.
+- `dst_account_type` (optional) default is `user`, the account type of dst account, can be set to `org` or `user`.
```
[the remark this answers is not in the document]
Comment (snowyu) [reply README_en.md:41]: Yes.

File: hub-mirror/hub.py [hunk 2]
```diff
@@ -72,16 +79,19 @@ def create_dst_repo(self, repo_name):
...
                 )
-                if response.status_code == 201:
+                result = response.status_code == 201
+                if result:
                     print("Destination repo creating accepted.")
                 else:
                     print("Destination repo creating failed: " + response.text)
         else:
             print(repo_name + " repo exist, skip creating...")
+        if result: time.sleep(2)
```
[the remark this answers is not in the document]
Comment (snowyu) [reply hub-mirror/hub.py:89]: I've met the error even if push again.

about the return result, It is not an incompatible modification. and I'm lazy man.

[the remark this answers is not in the document]
Comment (Yikun) [reply hub-mirror/hub.py:89]: The flake8 was failed by:
```
./hub.py:43:80: E501 line too long (81 > 79 characters)
./hub.py:89:18: E701 multiple statements on one line (colon)
./hub.py:94:80: E501 line too long (81 > 79 characters)
```

[the remark this answers is not in the document]
Comment (Yikun) [reply hub-mirror/hub.py]: > I've met the error even if push again.

You mean it was failed by without sleep 2s? Could you give the detail log or link in here?

> about the return result, It is not an incompatible modification.

Yeah, it works, but it doesn't solve the root issue I mentioned in previous comment, so if this fix (sleep 2s) it's not the specific for this PR(src/dst account pr), I prefer move this in a separate PR.

Then I will help to merge the src/dst account feature, and do a further dev on sleep 2s. WDYT?

**Comment by Yikun** [comment]
@snowyu Thanks for contribution, and also you could have a try on it, if there are any more issue on it, you could feel free to open  the new issue!

**Pull request merged**